test/gallery.test.js

```diff
@@ -109,7 +95,7 @@ describe('Applying Gallery Filters - allow only 1 value per filter', () => {
   test('if user selects invalid rating, it will not apply it', () => {
     let userSelectedFilters = { rating: [-5] }
     expect(generateFilterOptions(userSelectedFilters)).toHaveProperty('rating', null)
-    userSelectedFilters = { rating: 3 }
```

Comment:
Well this one is actually good. 
The point is that the input is invalid because it is not an array. 

It was specifically put as a valid number so that the test is even more relevant.

Reply:
Right, i complitely missed that... will fix.